client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/ContactsSection/ContactsTabsStyles.ts

```diff
@@ -0,0 +1,32 @@
+import { makeStyles } from '@material-ui/styles';
+import {Theme} from '@material-ui/core';
+
+const useStyles = makeStyles((theme: Theme) => ({
+    tab: {
+        flexDirection: 'row',
+        justifyContent: 'flex-start',
+    },
+    tabs: {
+        display: 'flex'
+    },
+    icon: {
+        '& svg': {
+            paddingLeft: '0.5vw',
+            flip: false,
+        }
+    },
+    selected: {
+        color: theme.palette.primary.main,
+    },
+    collapse: {
+        '@media screen and (max-width: 1700px)': {
+            width: '84%'
```

Comment:
we usually use vw vh, so why using precents?

Reply:
pushed fixed to be VW, but i think percentage here is better since its relative to parent... so if the parent width changes, the collapse width stays relative to the parent and not entire screen